tests/test_foreign_keys.py

```diff
@@ -1,11 +1,13 @@
-from nose.tools import assert_equal, assert_false, assert_true
+from nose.tools import assert_equal, assert_false, assert_true, raises
 from datajoint.declare import declare
+from datajoint import DataJointError
 
 from . import schema_advanced
 
 context = schema_advanced.schema.context
 
 
+@raises(DataJointError)     # TODO: remove after fixing issue #300
```

Comment:
would it be better to add a ```raise SkipTest('fix after issue 300')``` for these? 

That would theoretically make the 'brokenness' of the test slightly more visible so this isn't lost..

Either way, I've added a note in issue 300 itself.

Reply:
yes, thanks.  I just learned about `SkipTest`.

Reply:
actually, I do want to test that it throws a DataJoint error when an attempt to cascade delete across renamed foreign keys until #300 is fixed.  So I will keep it unchanged.

Reply:
Yes keep this in until #300 is fixed. Until then, the right behavior is to throw `DataJointError`. (but `SkipTest` is good to know - I didn't know about it)